libs/use-multi-media-streams/src/index.ts

```diff
@@ -1,56 +1,56 @@
-import { useState, useEffect, useReducer } from 'react';
+import { useEffect, useReducer, useRef } from 'react';
 
```

Comment:
Please read the issue target, we should remove `use-multi-media-streams` hook,  each source should manage it's own media stream then App collect all of the streams and manage them together.  Let's move this hook to app level.